RELEASE.md

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: minor
+
+Automatically cycle S3 objects in VHS to the Standard-IA storage class after 30 days.
+This reduces the cost of running a VHS.
+
+You can disable this behaviour by setting `cycle_objects_to_standard_ia = false`.
```

Comment:
Would you say this is minor given that it will change current behaviour?

Reply:
By the rules of semver, sure. It’s adding functionality in a backwards-compatible manner. Nothing in semver says that behaviour can’t change between versions.